src/Sarif.Multitool/Rules/RuleId.cs

```diff
@@ -19,5 +19,6 @@ public static class RuleId
         public const string InvalidIndex = "SARIF1017";
         public const string InvalidUriInOriginalUriBaseIds = "SARIF1018";
         public const string RuleIdMustBePresentAndConsistent = "SARIF1019";
+        public const string ReferToLatestSchema = "SARIF1020";
```

Comment:
>ReferToLatestSchema [](start = 28, length = 19)

I'm sorry, after all that I was inconsistent with the naming :(. Please use `ReferToFinalSchema`.

Reply:
Unfortunately that means renaming the rule class and the file that contains it, again. Sorry again.

---
In reply to: [433995091](https://github.com/microsoft/sarif-sdk/pull/1892#discussion_r433995091) [](ancestors = 433995091)

Reply:
no problem. i'm updating right now :)